src/vellum/workflows/nodes/displayable/tool_calling_node/utils.py

```diff
@@ -31,6 +33,30 @@
 CHAT_HISTORY_VARIABLE = "chat_history"
 
 
+class FunctionCallNodeMixin:
+    """Mixin providing common functionality for nodes that handle function calls."""
+
+    function_call_output: List[PromptOutput]
+    state: Any  # Will be properly typed by the inheriting node class
```

Comment:
[blocking] I don't believe we need to define this? Am worried about some inference magic we do if this is defined as an `Any`